### PR TITLE
Compresses png icons

### PR DESCRIPTION
**Pull request by SergeyZhukovsky**
Resolves https://github.com/brave/brave-browser/issues/25003

New size of a whole folder with images is 3.2Mb comparing to 19.4Mb that we had before that PR.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,17 +1,23 @@
 {
   "name": "token-lists",
-  "version": "1.1.8",
+  "version": "1.1.9",
   "description": "Manages custom token lists for Brave Wallet",
   "dependencies": {
     "@metamask/contract-metadata": "git+https://git@github.com/MetaMask/contract-metadata.git",
     "@solana/spl-token-registry": "git+https://git@github.com/brave/solana-token-list.git",
+    "imagemin": "^8.0.1",
+    "imagemin-pngquant": "^9.0.2",
```
Comment (diracdeltas) [inline package.json:9]: should these be added in devDependencies instead?

File: package.json [hunk 2]
```diff
@@ -1,17 +1,23 @@
...
     "qyu": "^0.4.7",
     "sharp": "^0.30.5",
     "svg-resizer": "git+https://git@github.com/brave/svg-resizer.git"
   },
+  "resolutions": {
+    "semver-regex": ">=4.0.5"
+  },
   "main": "scripts/index.js",
   "scripts": {
+    "preinstall": "npx npm-force-resolutions",
```
Comment (diracdeltas) [inline package.json:20]: do we need this, or can https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides be used instead?

**Comment by diracdeltas** [comment]
can we add `npm audit` somewhere in https://github.com/brave/token-lists/runs/8120285388?check_suite_focus=true so it runs as part of the PR actions and shows the results?

also please add some sanitization for the `source` variable in https://github.com/brave/token-lists/pull/17/files#diff-dc4ea81d99fc44c52c4d77fd990135768c119a76f33fbb95f5e5ed2688c8308cR76 to guard against shell injection; then we can dismiss https://github.com/brave/token-lists/security/code-scanning/2

https://socket.dev/npm/package/pngquant-bin/issues/6.0.1?tab=dependencies has some issues. probably the most concerning is that it depends on a uuid package which is deprecated:
Screenshot 2022-08-31 at 1 01 20 PM

can we run make sure these scripts only get run in a sandboxed environment (like a docker container that gets destroyed after outputting the compressed PNG icons)? that way if there are malicious images or vulns in the dependencies, it shouldn't compromise any machines.

**Comment by bbondy** [comment]
Just an FYI for security that the package which this repo produces is just JSON files and png files and does not include the dependencies at all.

**Comment by SergeyZhukovsky** [comment]
1, 2 and 3 is done and review suggestions are addressed. I created an issue to figure out 4 here https://github.com/brave/token-lists/issues/20
@diracdeltas could you please take another look on that PR?

**Comment by SergeyZhukovsky** [comment]
4 is done as well https://github.com/brave/token-lists/pull/24

**Comment by SergeyZhukovsky** [comment]
Security concerns are handled, merging it.

**Pull request merged**